docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
```

Comment:
Can you remove this copyright line? I know it's in the Tern's Dockerfile but that's because we created the Dockerfile when the project was still under the VMware org. Since tern is now apart of the LF we just need the license identifier in the file headers.

Reply:
You can also remove the VMware copyright statements from the other files in this PR. The SPDX License Identifier lines can stay.

Reply:
Removed copyright lines from web/app.py, docker_run.sh, and docker/Dockerfile.